package.json

```
@@ -21,13 +21,14 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./index.d.ts",
       "import": {
         "types": "./esm/index.d.mts",
         "default": "./esm/index.mjs"
       },
-      "module": "./esm/index.js",
-      "default": "./index.js"
+      "require": {
+        "types": "./index.d.ts",
+        "default": "./index.js"
+      }
```

Please don't remove "module" yet. (We will in v5.)
Please keep "default".

Does this work?
```suggestion
      "module": {
        "types": "./index.d.ts",
        "default": "./esm/index.js"
      },
      "default": {
        "types": "./index.d.ts",
        "default": "./index.js"
      }
```

---

@dai-shi let me look into that. BTW, you can also run `publint` locally -> `npx publint` under `dist/` directory

---

@dai-shi looks good